### PR TITLE
User can login

### PR DESCRIPTION
**Pull request by robin-lillqvist**
# [PT Story](https://www.pivotaltracker.com/story/show/172047598)
## User Story
```
As a Subscriber
In order to access the premium content
I would like to log in
```
## Changes proposed in this pull request:
* Adds login functionality
* Color theme for mobile app
## What I have learned working on this feature:
* That Ionic needs certain wrapper tags in order to function as they should.
* All Ionic components can take various amounts of parameters
* How to interact with Light and Dark themes in variables.css file
* That Ionic Labs is sensitive to computer OS dark mode settings.
* We still don't know if the app will deploy in light or dark theme
## Packages/Gems added
- J-tockauth packages
- Ionic-lab
## Screenshots (Only if client side story)
Screenshot 2020-04-02 at 16 22 24
Screenshot 2020-04-02 at 16 22 16

### CODE REVIEW

File: src/Display.jsx [hunk 1]
```diff
@@ -1,33 +1,33 @@
-import React from "react";
-import { connect } from "react-redux";
-import { useEffect } from 'react'
-import DisplayArticles from "./components/DisplayArticles";
-import { fetchArticles } from "./state/actions/articleActions";
-import { bindActionCreators } from "redux";
-import DisplaySingleArticle from "./components/DisplaySingleArticle";
+import React from 'react'
+import { connect } from 'react-redux'
+// import { useEffect } from 'react'
```
Comment (kaylawoodbury) [inline src/Display.jsx]: Do we need this comment for later?

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
+
+const Footer = props => {
+  const dispatch = useDispatch()
+  const authenticated = useSelector(state => state.authenticated)
+  let button
+
+  if (authenticated) {
+    button = (
+      <IonButton
+        name='login'
```
Comment (kaylawoodbury) [inline src/components/Footer.jsx]: can we update this to logout?

File: src/components/Menu.jsx [hunk 3]
```diff
@@ -9,18 +9,20 @@ const Menu = props => {
...
     <IonSegment id='article-category' scrollable>
-      <IonSegmentButton
+      <IonSegmentButton 
         name='All News'
         id=''
         as={Link}
         to={{ pathname: '/' }}
-        onClick={handleItemClick}  
+        onClick={handleItemClick}
+
```
Comment (kaylawoodbury) [inline src/components/Menu.jsx]: looks like an extra here

File: src/components/UserLogin.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { onLogin } from '../modules/authentication'
+import { useDispatch, useSelector } from 'react-redux'
+import { IonContent, IonButton, IonInput, IonCard } from '@ionic/react'
+
+const Login = props => {
+  const dispatch = useDispatch()
+  const authenticated = useSelector(state => state.authenticated)
+  // const message = useSelector(state => state.message)
```
Comment (kaylawoodbury) [inline src/components/UserLogin.jsx]: do we need this comment?

**Review by kaylawoodbury — COMMENTED** [review]
Looks good!  Just left a couple comments.

**Comment by pierre-1** [comment]
Looks good, we just need to update the screenshots and logo file with the most recent client logo.

**Pull request merged**